app/views/wiki/_wikis.erb

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Hi @mittal-parth, shouldn't this file extension be `.html.erb`? I know everything still works when you use just `.erb` but it's a lot easier to know what kind of file you're dealing with when you include `html`. Thanks!

Reply:
Oops sorry, I don't how I did that wrong 🤦🏼‍♂️